### DISCUSSION TITLE
Plates and Spheres in Hard Body MC

### DISCUSSION THREAD

**Discussion by mdsimulationgroup** [glotzerlab/hoomd-blue#1929] [General]
Hello,

I am new to HOOMD-blue and wanted to find out if it would be possible to run hard body monte carlo on a system of platelets and spheres?  In the 2020 HOOMD-blue paper, there is this line:

"""
All particles in a simulation must be of the same shape class, but may be of different particle types, where each particle type has separate shape parameters.
"""

Which has me confused if I could mix general structure shapes, like platelets and spheres.

Thanks for the help! 
John

**Comment by joaander** [comment]
Use [`ConvexSpheroPolyhedron`](https://hoomd-blue.readthedocs.io/en/v4.9.1/module-hpmc-integrate.html#hoomd.hpmc.integrate.ConvexSpheropolyhedron).
